packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx

```diff
@@ -463,7 +462,6 @@ const DataGridProVirtualScroller = React.forwardRef<
                 minFirstColumn: leftRenderContext.firstColumnIndex,
                 maxLastColumn: leftRenderContext.lastColumnIndex,
                 availableSpace: 0,
-                ignoreAutoHeight: true,
                 rows: bottomPinnedRowsData,
                 rowIndexOffset: topPinnedRowsData.length + (mainRows ? mainRows.length : 0),
```

Comment:
Should we pass `position` here as well?
It seems to work with pinned rows, even though `position` is not passed: https://codesandbox.io/s/muddy-feather-j7m08i?file=/demo.tsx

Reply:
Done! It only worked in the example for a coincidence.